crates/extensions/az_mapper_ext/src/size_threshold.rs

```diff
@@ -0,0 +1,22 @@
+use tedge_mqtt_ext::MqttMessage;
```

Comment:
As we've already discussed, this module must be moved to a common lib (not under `extensions`) and reused by all mappers.

Reply:
moved into `mapper_utils` into the common directory.

Reply:
I see little benefits to have a crate to define this size-threshold.